edge-util/src/Microsoft.Azure.Devices.Edge.Util/ISharedAccessSignatureCredential.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Util
+{
+    /// <summary>
+    /// Credential interface used for authentication and authorization.
+    /// </summary>
+    internal interface ISharedAccessSignatureCredential
```

Comment:
do we have multiple implementations or use it for mocking?

Reply:
I took in the interface so we could do mocking where appropriate, if you would rather I remove this and we add it when necessary that can work too :)

Reply:
it's nit, but I'd rather avoid interfaces if we have only a single implementation. For mocking, I would mark methods as virtual.

Reply:
Removed :)